lib/asm-bcc.js

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2015, Matt Godbolt
```

Comment:
You're the author of this file even if it took lib/asm.js as a base. Feel free to update the copyright header :)

Reply:
I'm not one to write my name everywhere, but I won't blame Matt for my code ;-)